tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_Shape.cpp

```diff
@@ -146,26 +147,32 @@ std::vector<std::vector<std::complex<double>>> generate_random_coefs(
     size_t l_max, size_t m_max, gsl::not_null<std::mt19937*> generator) {
   std::vector<std::vector<std::complex<double>>> coefs{};
   // if the coefficients are made too large, the map has a good chance of
-  // mapping through the center, causing the test to fail
-  std::uniform_real_distribution<double> coef_dis{0.1, 0.2};
-  const auto ran = [&coef_dis, &generator]() { return coef_dis(*generator); };
+  // mapping through the center, causing the test to fail which is why we damp
+  // higher mode coefficients. The factor was found empirically by trial and
+  // error.
+  std::uniform_real_distribution<double> coef_dist{0., 1.};
 
   for (size_t l = 0; l <= l_max; ++l) {
-    std::vector<std::complex<double>> tmp{ran()};
+    // m=0 is real
+    std::vector<std::complex<double>> tmp{
+        make_with_random_values<double>(generator, coef_dist, 1) /
+        double(8 * l * l + 1)};
```

Comment:
what is this? :D Could it be the factor from the comment above?

Reply:
It is indeedy